core/adapters/base.ts

```diff
@@ -1,35 +1,39 @@
 import { dataform } from "@dataform/protos";
 
 export abstract class Adapter {
-  public where(query: string, where: string) {
-    return `select * from (${query}) as subquery
-        where ${where}`;
+  public abstract resolveTarget(target: dataform.ITarget): string;
+
+  public normalizeIdentifier(identifier: string) {
+    return identifier;
```

Comment:
What's the point of this function?

Reply:
It's to give a default implementation of this function (defined on the interface) for all other Adapters.